homepage/scripts/scripts.js

```diff
@@ -195,13 +195,21 @@ const getCookie = (name) => document.cookie
 async function imsCheck() {
   const { loadIms, setConfig } = await import(`${miloLibs}/utils/utils.js`);
   setConfig({ ...CONFIG, miloLibs });
+  let isSignedInUser = false;
   try {
     await loadIms();
+    if (window.adobeIMS?.isSignedInUser()) {
+      await window.adobeIMS?.validateToken();
```

Comment:
don't see the extra `?` to be needed here, but no harm in keeping it

Reply:
addressed in the follow-up PR: https://github.com/adobecom/homepage/pull/80